### DISCUSSION TITLE
Prevent ctrl+tab cycling from leaving current panel tabs

### DISCUSSION THREAD

**Discussion by quadsteel** [mbnuqw/sidebery#1229] [Q&A] [answered]
Is there a way to keep ctrl+tab tab cycling from going outside the current panel? As it is by default, if you keep hitting ctrl+tab after you reach the last tab on your panel (provided you're cycling tabs in order, not by recently used) it will switch to the first tab of the next panel an so on. Can we force it to keep cycling inside the same panel? Is there an option that I missed for it?

And on the other hand, if we keep it cycling to the next panel, can we have the sidebar also update to reflect the change in active panels?

**Comment by mbnuqw** [comment, marked as answer]
> Is there a way to keep ctrl+tab tab cycling from going outside the current panel?

Enable `Hide native tabs of inactive panels` option

> And on the other hand, if we keep it cycling to the next panel, can we have the sidebar also update to reflect the change in active panels?

Set `Automatically switch panel on activating tab of another panel` to `always` (v5 only, in v4 this should work out of the box)
